src/klein/interfaces.py

```diff
@@ -24,7 +24,7 @@
 )
 
 if TYPE_CHECKING:               # pragma: no cover
-    from ._storage.memory import MemorySessionStore, MemorySession
+    from .storage._memory import MemorySessionStore, MemorySession
```

Comment:
Ugh this  double-private thing just reminds me we really need to get `publication` in here at some point (not really germane to this change though).

Reply:
@glyph is talking about https://pypi.org/project/publication/, I think

Reply:
Yep, that’s the one.